node/service/src/chain_spec.rs

```diff
@@ -298,18 +297,23 @@ fn cere_dev_local_testnet_genesis(wasm_binary: &[u8]) -> cere_dev::RuntimeGenesi
 pub fn cere_dev_local_testnet_config() -> Result<CereDevChainSpec, String> {
 	let wasm_binary = cere_dev::WASM_BINARY.ok_or("Cere Dev development wasm not available")?;
 
+	#[allow(deprecated)]
 	Ok(CereDevChainSpec::from_genesis(
 		"Local Testnet",
 		"cere_dev_local_testnet",
 		ChainType::Local,
-		move || cere_dev_local_testnet_genesis(wasm_binary),
+		cere_dev_local_testnet_genesis,
 		vec![],
 		None,
 		Some(DEFAULT_PROTOCOL_ID),
 		None,
 		None,
 		Default::default(),
+		wasm_binary,
 	))
+
+	//TODO: Migrate to builder
```

Comment:
@Aideepakchaudhary please remove TODO. This `fn` migration will be addressed on v1.7 upgrade.